hack/tilt/deploy.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+TILT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+SCRIPT_DIR=$(dirname $TILT_DIR)
+SRC_DIR=$(dirname $SCRIPT_DIR)
+BIN_DIR=$(dirname $SCRIPT_DIR)/temp/tilt/bin 
+
+# system data
+NAMESPACE="kube-system"
```

Comment:
should these be in the tilt.dev instead

Reply:
We'll use this namespace to deploy `kpng` DaemonSet. We may change it to `tilt.dev`

Reply:
+1